### PR TITLE
chore: add new k8s releases to VHD CI spec

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

Add the following new Kubernetes versions to the VHD spec:

- 1.16.0
- 1.15.4
- 1.13.11
- 1.14.7

For AKS, AKS Engine, and Azure Stack.

Also deprecate VHD support for 1.11 and 1.12 versions on Azure Stack.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -401,9 +400,11 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
       HYPERKUBE_URL="mcr.microsoft.com/k8s/azurestack/core/hyperkube-amd64:v${KUBERNETES_VERSION}"
     else
       HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
-      CONTAINER_IMAGE="k8s.gcr.io/cloud-controller-manager-amd64:v${KUBERNETES_VERSION}"
-      pullContainerImage "docker" ${CONTAINER_IMAGE}
-      echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+      if (( $(echo ${KUBERNETES_VERSION} | cut -d"." -f2) < 16 )); then
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: the ident seems off here

Comment (jackfrancis) [reply packer/install-dependencies.sh]: do you mean because 2 chars instead of 4?

Comment (CecileRobertMichon) [reply packer/install-dependencies.sh]: yeah, just that the `fi` and the `if` aren't aligned

**Comment by jackfrancis** [comment]
@CecileRobertMichon there were a mix of tabs and spaces inside that block, addressed here:

https://github.com/Azure/aks-engine/pull/1992

**Pull request merged**